app/models/config.rb

```diff
@@ -1,6 +1,6 @@
 class Config < ActiveRecord::Base
   belongs_to :project
-  scope :for, ->(metric_name) { where(:metric_name => metric_name).limit(1) }
+  scope :for, ->(metric_name) { where(:metric_name => metric_name).first }
```

Comment:
this isn't the same thing.  my original version of this scope retrieves a single record.  yours theoretically retrieves multiple records and then throws away all but the first.  (technically there should only be one, so maybe i'm being pedantic)